pyproject.toml

```diff
@@ -46,7 +52,7 @@ embeddings-sagemaker = ["boto3"]
 vector-stores-qdrant = ["llama-index-vector-stores-qdrant"]
 vector-stores-chroma = ["llama-index-vector-stores-chroma"]
 vector-stores-postgres = ["llama-index-vector-stores-postgres"]
-
+storage-postgres = ["llama-index-storage-docstore-postgres","llama-index-storage-index-store-postgres","psycopg2-binary","asyncpg"]
```

Comment:
I was thinking last night.   Might we want to call this `nodestore-postgres` for consistency too?

Reply:
We've tried to stick with LlamaIndex notation: 
llama-index-vector-stores-chroma -> vector-stores-postgres

In this case, given we are aggregating both docstore and index-store, I think the most correct way would be to call it `storage-nodestore-postgres` (given "lama-index-storage-docstore-postgres","llama-index-storage-index-store-postgres)